Formula/migrate.rb

```diff
@@ -0,0 +1,28 @@
+class Migrate < Formula
+  desc "Database migrations CLI tool"
+  homepage "https://github.com/golang-migrate/migrate"
+  url "https://github.com/golang-migrate/migrate.git",
+      :tag => "v3.3.0",
+      :revision => "bcd996f3df28363f43e2d0935484c4559537a3eb"
+  revision 1
+
+  depends_on "dep"
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/golang-migrate/migrate").install buildpath.children
+
+    # Build and install cli as "migrate"
+    binname = "migrate"
+    cd "src/github.com/golang-migrate/migrate" do
+      system "dep", "ensure", "-vendor-only"
+      system "make", "build-cli"
+      bin.install "cli/build/migrate.darwin-amd64" => binname
+    end
+  end
+
+  test do
+    assert_match "v3.3.0", shell_output("#{bin}/migrate -version")
```

Comment:
Any chance of a test that does more with functionality here?

Reply:
migrate interacts with db migration files, I can't came up with a case to test functionality yet...

Reply:
thanks for the thorough review! I've submitted other changes in 4e5f226, not sure why revision line is not marked as outdated though.

Reply:
revision line removed

Reply:
Agreed with @DomT4. This needs to be a functional test not just a version test.